tests/integration/helm_otelcol_events_test.go

```diff
@@ -1,95 +0,0 @@
-package integration
-
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
-
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"sigs.k8s.io/e2e-framework/pkg/envconf"
-	"sigs.k8s.io/e2e-framework/pkg/features"
-
-	"github.com/gruntwork-io/terratest/modules/k8s"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-
-	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal"
-	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/ctxopts"
-	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/stepfuncs"
-)
-
-func Test_Helm_Otelcol_Events(t *testing.T) {
-	const (
-		tickDuration                     = 3 * time.Second
-		waitDuration                     = 5 * time.Minute
-		expectedEventCount          uint = 50 // number determined experimentally
-		expectedSecretEndpointCount int  = 1
-	)
-
-	featInstall := features.New("installation").
-		Assess("sumologic secret is created with endpoints",
-			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
-				k8s.WaitUntilSecretAvailable(t, ctxopts.KubectlOptions(ctx), "sumologic", 60, tickDuration)
-				secret := k8s.GetSecret(t, ctxopts.KubectlOptions(ctx), "sumologic")
-				require.Len(t, secret.Data, expectedSecretEndpointCount, "Secret has incorrect number of endpoints. There should be only 1 endpoint, for events.")
-				return ctx
-			}).
-		Assess("otelcol events statefulset is ready",
-			stepfuncs.WaitUntilStatefulSetIsReady(
-				waitDuration,
-				tickDuration,
-				stepfuncs.WithNameF(
-					stepfuncs.ReleaseFormatter("%s-sumologic-otelcol-events"),
-				),
-				stepfuncs.WithLabelsF(
-					stepfuncs.LabelFormatterKV{
-						K: "app",
-						V: stepfuncs.ReleaseFormatter("%s-sumologic-otelcol-events"),
-					},
-				),
-			),
-		).
-		Assess("otelcol events buffers PVCs are created",
-			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
-				namespace := ctxopts.Namespace(ctx)
-				releaseName := ctxopts.HelmRelease(ctx)
-				kubectlOptions := ctxopts.KubectlOptions(ctx)
-
-				t.Logf("kubeconfig: %s", kubectlOptions.ConfigPath)
-				cl, err := k8s.GetKubernetesClientFromOptionsE(t, kubectlOptions)
-				require.NoError(t, err)
-
-				assert.Eventually(t, func() bool {
-					pvcs, err := cl.CoreV1().PersistentVolumeClaims(namespace).
-						List(ctx, v1.ListOptions{
-							LabelSelector: fmt.Sprintf("app=%s-sumologic-otelcol-events", releaseName),
-						})
-					if !assert.NoError(t, err) {
-						return false
-					}
-
-					return err == nil && len(pvcs.Items) == 1
-				}, waitDuration, tickDuration)
-				return ctx
-			}).
-		Feature()
```

Comment:
there is no need to check it anymore?

Reply:
This check is present in the default test.